README.md

```diff
@@ -1,7 +1,25 @@
-# sidecar-proxy
-A reverse proxy, meant to run in a container alongside a service it needs to serve (in the same pod).
+# Sidecar proxy
+A container meant to run alongside a service (in the same pod) in order to serve Prometheus metrics for that service.
 
-Supports both HTTP (Go's built in ReverseProxy) and Websocket (https://github.com/koding/websocketproxy) requests.
-In addition, serves Prometheus type metrics through the `/metrics` endpoint.
+Supported metrics:
+* `num_of_requests` - prometheus `CounterVec` that simply counts requests using a reverse proxy. 
+Supports both HTTP (Go's built in ReverseProxy) and Websocket 
+(https://github.com/koding/websocketproxy) requests<br>
+
+* `jupyter_kernel_busyness` - prometheus `GaugeVec` that is set to 1 if Jupyter has one or more busy kernels, and to 0 
+otherwise.
+Periodically queries Jupyter's `/api/kernels` endpoint
+
+The container includes a server that serves Prometheus metrics through the `/metrics` endpoint.
+
+All metrics contain these labels: `namespace`, `service_name`, `instance_name`.
+
+The code was built so it will be easy to extend it and add new metrics. it's simply done by creating a new metrics 
```

Comment:
```suggestion
The code was built so it will be easy to extend it and add new metrics. This is  performed by creating new metric handlers that implement ...
```